CIP-9999/README.md

```diff
@@ -0,0 +1,189 @@
+Created: 2009-02-14
+---
+```
+</details>
+
+<details>
+  <summary><span id="repository-organization"><strong>Repository Organization</strong><span></summary>
+
+A CPS must be stored in a specific folder named after its number and in a file called `README.md`. Before a number is assigned, use `????` as a placeholder name.
+
+Additional supporting files (such as diagrams, binary specifications, dialect grammars, JSON schemas etc.) may be added to the CPS's folder under freely chosen names.
+
+For example:
+
+```
+CPS-0001
+├── README.md
+└── requirements.toml
+```
+</details>
+
+> **Note** A reference template is available in [.github/CPS-TEMPLATE.md][CPS-TEMPLATE.md]
+
+#### Statuses
+
+From its creation onwards, a problem statement evolves around the following statuses.
+
+Status       | Description
+---          | ---
+Open         | Any problem statement that is fully formulated but for which there still exists no solution that meets its goals. Problems that are only partially solved shall remain _'Open'_ and list proposed solutions so far in their header's preamble.
+Solved       | Problems for which one or more solutions have been found and implemented. When solved via a CIP, the solved status should indicate it as such: `Solved: by <CIP-XXXX>[,<CIP-YYYY>,...]`.
+Inactive    | The statement is deemed obsolete or withdrawn for another reason. A short reason must be given between parentheses. For example: `Inactive (..._reason_...).
+
+> **Note** There is no "draft" status: a proposal which has not been merged (and hence exists in a PR) is a draft CPS. Draft CPSs should include the status they aim for on acceptance, typically but not always; this will be _'Open'_.
+
+#### Categories
+
+As defined in [CIP-0001][].
+
+### The CPS Process
+
+#### 1. Early stages (same as CIP-0001)
+
+##### [1.a. Authors open pull requests with their problem statement](https://cips.cardano.org/cips/cip1#authors-a-open-pull-requests)
+
+##### [1.b. Authors seek feedback](https://cips.cardano.org/cips/cip1#authors-seek-feedback)
+
+#### 2. Editors' role (same as CIP-0001)
+
+##### [2.a. Triage in bi-weekly meetings](https://cips.cardano.org/cips/cip1##triage-in-bi--weekly-meetings)
+
+##### [2.b. Reviews](https://cips.cardano.org/cips/cip1#reviews)
+
+#### 3. Merging CPSs in the repository
+
+A statement must be well-formulated (i.e. unambiguous) and demonstrate an existing problem (for which use cases exist with no suitable alternatives). When related to a current project, the problem statement must also have been acknowledged by its respective code owners. In some cases, problem statements may be written after the facts and be merged directly as _'Solved'_ should they document in more depth what motivated an existing solution.
+
+Problem statements deemed unclear, for which alternatives exist with no significant drawbacks or establish unrealistic goals, shall be rejected (i.e. pull request closed without merge) with justifications or withdrawn by their authors.
```

Comment:
Do we want to explicitly allow for the possibility of having multiple competing presentations of a problem? I guess we can leave it up to the editors.